src/test/java/TypewiseAlert/TypewiseAlertTest.java

```diff
@@ -10,6 +10,49 @@ public class TypewiseAlertTest
     public void infersBreachAsPerLimits()
     {
       assertTrue(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+        BreachTypes.TOO_LOW);
+    }
+    
+    @Test
+    public void infersBreachAsPerHighLimits()
+    {
+      assertTrue(TypewiseAlert.inferBreach(35, 20, 30) ==
+    		  BreachTypes.TOO_HIGH);
+    }
+    
+    @Test
+    public void infersBreachAsPerNormalLimits()
+    {
+      assertTrue(TypewiseAlert.inferBreach(25, 20, 30) ==
+    		  BreachTypes.NORMAL);
+    }
+    
+    //================================================
+    
+    @Test
+    public void checkAlert_controller_Passive_cooling() {
+        assertTrue(TypewiseAlert.checkAndAlert(AlertTargets.TO_CONTROLLER,CoolingTypes.PASSIVE_COOLING,31));
+    }
+
+    @Test
+    public void checkAlert_controller_Med_cooling() {
```

Comment:
How do we verify if the intended target has received the alerts? there should be a test to verify that too...